app/controllers/accountants_controller.rb

```diff
@@ -7,24 +7,34 @@ class AccountantsController < ApplicationController
   def index
     patients = pledged_patients.includes(:clinic)
     @patients = paginate_results(patients)
+
+    @entry_name = t('common.patient')
   end
 
   def search
+    index
```

Comment:
Rails controllers (and this is true for stuff like express.js, django, etc too I think) do a lot of behind the scenes stuff, so it's not advisable to call endpoint methods from other endpoint methods. What are we trying to accomplish here? If it's complicated let's roll it up into a private method that both index and search endpoints can share.

Reply:
ah, that's fair. we need access to `@patients` in `search` to know how many total patients there are, for Kaminari interface display. Maybe I'll make a new method for that setup stuff.